docs/integrations/builtin/app-nodes/n8n-nodes-base.telegram.md

```diff
@@ -18,33 +18,629 @@ Refer to [Telegram credentials](/integrations/builtin/credentials/telegram/) for
 
 ## Operations
 
-* Chat
-    * Get
-    * Get Administrators
-    * Get Member
-    * Leave
-    * Set Description
-	* Set Title
-* Callback
-    * Answer Query
-    * Answer Inline Query
-* File
-    * Get
-* Message
-    * Delete Chat Message
-    * Edit Chat Message
-    * Pin Chat Message
-    * Send Animation
-    * Send Audio
-    * Send Chat Action
-    * Send Document
-    * Send Location
-    * Send Media Group
-    * Send Message
-    * Send Photo
-    * Send Sticker
-    * Send Video
-    * Unpin Chat Message
+* Chat operations
```

Comment:
```suggestion
* **Chat** operations
```
What do you think about making the resource names (Chat, Callback, File, Message) bold since they're also an UI option?